python/ray/tests/test_placement_group_5.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import sys
+import ray
+
+from ray.util.client.ray_client_helpers import connect_to_client_or_not
+
+
+def are_pairwise_unique(g):
+    s = set()
+    for x in g:
+        if x in s:
+            return False
+        s.add(x)
+    return True
+
+
+@ray.remote
+class Actor(object):
+    def __init__(self):
+        self.n = 0
+
+    def value(self):
+        return self.n
+
+
+def index_to_actor(pg, index):
```

Comment:
Can you move these functions/vars inside `test_placement_group_bin_packing_priority `? This doesn't need to be reused